packages/common/src/ui/components/modals/AlertModal/AlertModalContext.tsx

```diff
@@ -0,0 +1,28 @@
+import { createContext } from 'react';
+
+export interface AlertModalState {
+  open: boolean;
+  message: string;
+  title: string;
+  iconType?: 'alert';
+  onOk?: () => void;
+}
+
+export interface AlertModalControllerState extends AlertModalState {
+  setState: (state: AlertModalState) => void;
+  setOpen: (open: boolean) => void;
+  setMessage: (message: string) => void;
+  setTitle: (title: string) => void;
+  setOnOk: (onOk: () => void) => void;
+}
+
+export const AlertModalContext = createContext<AlertModalControllerState>({
+  open: false,
+  message: '',
+  title: '',
+  setOpen: () => {},
+  setState: () => {},
+  setMessage: () => {},
+  setTitle: () => {},
+  setOnOk: () => {},
+});
```

Comment:
😁  

I find the `useConfirmationModal` hook incredibly useful, versatile and easy - thought to do the same thing with the alert modal